src/components/AddButton.js

```diff
@@ -0,0 +1,53 @@
+import React,  { Component, } from "react";
+import PropTypes from "prop-types";
+import { Button, View, TouchableOpacity, Text, Image} from 'react-native';
+import styled from 'styled-components';
+import AddIcon from '../assets/images/icons/add-icon.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
import AddIcon from "../assets/images/icons/add-icon.png";
```